app/src/main/java/com/doctor/yumyum/presentation/ui/mypage/MyPageFragment.kt

```diff
@@ -1,28 +1,78 @@
 package com.doctor.yumyum.presentation.ui.mypage
 
+import android.content.Intent
 import android.os.Bundle
-import android.view.LayoutInflater
 import android.view.View
-import android.view.ViewGroup
 import androidx.lifecycle.ViewModelProvider
 import com.doctor.yumyum.R
 import com.doctor.yumyum.common.base.BaseFragment
+import com.doctor.yumyum.common.utils.gradePoint
 import com.doctor.yumyum.databinding.FragmentMyPageBinding
+import com.doctor.yumyum.presentation.ui.taste.TasteActivity
+import kotlinx.coroutines.CoroutineScope
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.launch
 
 class MyPageFragment : BaseFragment<FragmentMyPageBinding>(R.layout.fragment_my_page) {
 
-    private lateinit var viewModel: MyPageViewModel
+    private val viewModel by lazy {
+        ViewModelProvider(
+            this,
+            ViewModelProvider.NewInstanceFactory()
+        )[MyPageViewModel::class.java]
+    }
 
-    override fun onCreateView(
-        inflater: LayoutInflater, container: ViewGroup?,
-        savedInstanceState: Bundle?
-    ): View? {
-        return super.onCreateView(inflater, container, savedInstanceState)
+    private val gradeBadge: Map<String, Int> by lazy {
+        mapOf(
+            getString(R.string.grade_student) to R.drawable.ic_badge_student,
+            getString(R.string.grade_bachelor) to R.drawable.ic_badge_bachelor,
+            getString(R.string.grade_master) to R.drawable.ic_badge_master,
+            getString(R.string.grade_expert) to R.drawable.ic_badge_expert,
+            getString(R.string.grade_professor) to R.drawable.ic_badge_professor,
+        )
+    }
+    private val nextGrade: Map<String, String> by lazy {
+        mapOf(
+            getString(R.string.grade_student) to getString(R.string.grade_bachelor),
+            getString(R.string.grade_bachelor) to getString(R.string.grade_master),
+            getString(R.string.grade_master) to getString(R.string.grade_expert),
+            getString(R.string.grade_expert) to getString(R.string.grade_professor),
+            getString(R.string.grade_professor) to getString(R.string.grade_professor),
+        )
     }
 
     override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
         super.onViewCreated(view, savedInstanceState)
-        viewModel = ViewModelProvider(this).get(MyPageViewModel::class.java)
         binding.viewModel = viewModel
+        binding.lifecycleOwner = viewLifecycleOwner
+
+        CoroutineScope(Dispatchers.IO).launch {
+            viewModel.getUserInfo()
+        }
+
+        binding.apply {
+            myPageTvTasteSetting.setOnClickListener {
+                val tasteIntent = Intent(context, TasteActivity::class.java)
+                tasteIntent.putExtra(getString(R.string.taste_mode), true)
+                startActivity(tasteIntent)
+            }
+        }
```

Comment:
좀 더 간단하게 표현할 수 있을 것 같습니다~!
```suggestion
        binding.myPageTvTasteSetting.setOnClickListener {
            val tasteIntent = Intent(context, TasteActivity::class.java)
            tasteIntent.putExtra(getString(R.string.taste_mode), true)
            startActivity(tasteIntent)
        }
```

Reply:
다른 `clickListener`들도 `apply`문 안에 밀어넣으려고 했습니다~!